src/Microsoft.ML.TimeSeries/SrCnnEntireTransformer.cs

```diff
@@ -0,0 +1,910 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Microsoft.ML;
+using Microsoft.ML.CommandLine;
+using Microsoft.ML.Data;
+using Microsoft.ML.Internal.Utilities;
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Transforms.TimeSeries;
+
+[assembly: LoadableClass(typeof(SrCnnEntireTransformer),
+        typeof(SrCnnEntireTransformer), null, typeof(SignatureLoadModel),
+        "SrCnnEntire Transformer",
+        SrCnnEntireTransformer.LoaderSignature)]
+namespace Microsoft.ML.Transforms.TimeSeries
+{
+    public sealed class SrCnnEntireTransformer : OneToOneTransformerBase
```

Comment:
>OneToOneTransformerBase [](start = 49, length = 23)

Now that this transformer has two input columns and one output column, it cannot derive from `OneToOneTransformerBase`.